### DISCUSSION TITLE
Questions about the Inputevents table [MIMIC-IV]

### DISCUSSION THREAD

**Discussion by Qinyu-Allen-Zhao** [MIT-LCP/mimic-code#1141] [MIMIC-IV] [answered]
Hi!

I have some questions about the Inputevents table. It'll be really appreciated if you could help me.

(1) I noticed that there is a column called statusdescription in the Inputevents table. [The document](https://mimic.mit.edu/docs/iv/modules/icu/inputevents/) said "STATUSDESCRIPTION states the ultimate status of the item, or more specifically, row." But I feel it's kind of hard to understand this statement.

Does it mean that the labeled status was what happened after the endtime of that row? For example, if the status was "Changed", was the delivery changed after the end time? Or it might be changed at any time point between starttime and endtime?

(2) Another question is that I find the ORDERCATEGORYDESCRIPTION of many rows are labeled as continuous, but the endtime - starttime is 1. According to the document, "A bolus will be listed as ending one minute after it started, i.e. ENDTIME: STARTTIME + 1 minute". I'm wondering whether those rows are bolus or continuous.

![image](https://user-images.githubusercontent.com/71951741/128603364-38cae05e-ebfb-4bb0-a298-a3b94a7d668e.png)

(3) I found that some records in inputevents are not in emar table or prescriptions table. Were these drugs administrated without prescriptions? If those tables have different records, which one should I use?

Thank you very much!!!!

Qinyu

**Comment by SichengH** [comment, marked as answer]
(1) Row means each row of data, the same meaning as the word item. 

```
Changed - The current delivery has ended as some aspect of it has changed (most frequently, the rate has been changed)
```

So I would say at the end. 

(2) Basically, bolus, non-iv meds, and drugs don't have duration information. 1 is like a placeholder, doesn't do indication the actual duration. 

(3) Inputevents are documented specifically for ICU patients; emar and prescription are for all hospital patients and from a different information system. Unfortunately, there are not 100% matched. So which one to use really depend on what information you are trying to get. If you would like to investigate fluids given in the ICU, use inputevents.

**Reply by Qinyu-Allen-Zhao** [reply]
Hi! Thank you for your reply!

I'm still kind of confused about the second question. Indeed, "bolus, non-iv meds, and drugs don't have duration information. "
But some rows are labeled as continuous-IV but have a length of 1 minute. As shown in my figure, the minimum duration of "Continuous IV" is also 1 minute. How do we understand that?

Many thanks!!

**Reply by SichengH** [reply]
Since those drugs are often given in a short time period, the time and rate variable wasn't documented the same as other IV drugs. This is just how the data is documented. The time you calculated doesn't mean the drug is given in exactly 1 minute, I would keep starttime or charttime and ignore the endtime and rate, also keep the volume.

**Reply by Qinyu-Allen-Zhao** [reply]
Thank you for your kind reply!
